_sass/_base.scss

```diff
@@ -670,4 +670,25 @@ pre {
   .entry:last-child {
     border-bottom: none;
   }
-}
\ No newline at end of file
+}
+
+.ehcache-documentation {
```

Comment:
Couldn't you have moved the common attributes (float, font-size) up to the parent scope?